### PR TITLE
HTML Validation and other things

### PR DESCRIPTION
**Pull request by aayn**
Sorry. Though I understood squashing more or less, I ended up following the easy path of removing '.git' and starting again :P

### CODE REVIEW

**Comment by polybuildr** [comment]
We'll review this again, but could you please update the commit message title and body to something else please? :P

**Comment by aayn** [comment]
Sure :P

**Comment by aayn** [comment]
Done.

**Comment by polybuildr** [comment]
Awesome. :)

Again, I didn't go through this in detail. Looked through which files were included though, and overall LGTM. @ParthKolekar will decide whether this is merge-worthy.

Meanwhile, could you please add an `index.html` file to the root of the repo which is a valid HTML document and includes:

``` html
<meta http-equiv="refresh" content="0; url=./main/">
```

or something equivalent? (Not sure about the `url=` field, please test once.)

This will ensure that someone opening up `/13` (once we host this on felicity.iiit.ac.in), won't get a 404, but instead will get redirected to the right URL.

However, if you feel that this should be done in another commit and PR, feel free to say so, and we'll do it that way.

**Comment by ParthKolekar** [comment]
@polybuildr 

I will not accept a 

```
<meta http-equiv="refresh" content="0; url=./main/">
```

You will need to bring it to /13

**Comment by polybuildr** [comment]
@ParthKolekar that doesn't make any sense. The original site was at `/13/main/`. Why would be want it to be at `/13`?

File: Misc/sponsors/index.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+<li style="width: 300px; height: 186px;" class="">
+	<img src="images/1_ms.jpg" width="210" height="130" alt="3_sbh.jpg">
```
Comment (ParthKolekar) [inline Misc/sponsors/index.html:184]: when MS alt text is SBH.

#YOLO

File: main/index.html [hunk 2]
```diff
@@ -0,0 +1,1415 @@
...
+
+
+<!-- Felicty header -->
+        <div class="unit static threeWide" id="Logo" style="height: 120px">
+
+
+
+
+ <div>
+    <marquee  id="marqueedisp" scrollamount="3" onmouseout="this.scrollAmount=3" onmouseover="this.scrollAmount=0" direction="left" class="list_marquee" >
```
Comment (ParthKolekar) [inline main/index.html:123]: I don't like feli.co links.

@polybuildr  will decide what to do here.

File: main/js/get_answer_GHJGDJHGJDHG.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    answer_box.innerHTML = "<p>Hey, ask me a question!</p>";
+}
+
+
+function get_answer(){
+    question_box = document.getElementById("question_box");
+    question = encodeURIComponent(question_box.value);
+    //question = question_box.value;
+    
+$.ajax({type:'GET',url:'http://10.4.3.65/web2py/ChatBot_V4/default/get_answer',success:function(temp){
```
Comment (ParthKolekar) [inline main/js/get_answer_GHJGDJHGJDHG.js:20]: YOLO?

Hardcoded IP. 
:grin:

**Comment by ParthKolekar** [comment]
@polybuildr 

¯_(ツ)_/¯

Fine then. 

meta tag it is.

File: main/index.html (hunk 2 above)
Comment (polybuildr) [inline main/index.html:123]: I'd say we let them 404, at least for this commit. We'll try to come up with a plan for them later - it's too time consuming and we may not get anywhere finally.

File: main/js/get_answer_GHJGDJHGJDHG.js (hunk 3 above)
Comment (polybuildr) [inline main/js/get_answer_GHJGDJHGJDHG.js:20]: What do we do about this line? I don't think we want IPs entering the repo.

**Comment by polybuildr** [comment]
{{ parth_emo }}

**Comment by aayn** [comment]
Done.

**Comment by polybuildr** [comment]
Okay. I'm happy with this now, even though I haven't inspected in much detail.

@ParthKolekar, feel free to merge once your concerns are resolved.

File: main/js/get_answer_GHJGDJHGJDHG.js (hunk 3 above)
Comment (aayn) [inline main/js/get_answer_GHJGDJHGJDHG.js:20]: That IP appears in 3 places. You guys decide what to do with it. I'll leave it for now.

**Comment by ParthKolekar** [comment]
Merged.

If changes are required, @aayn can always make another PR.

**Comment by polybuildr** [comment]
We have our first proper merge. :D

Great work, @aayn. :)

@ParthKolekar, yeah, he can.

I forgot to say {fixes T12}. Now the "merged" message won't come on phab. :'(

**Comment by hharchani** [comment]
Why don't you people swap contents of /index.html and main/index.html

**Comment by polybuildr** [comment]
Because then we'll probably have to update a lot of URL references, and we'll also lose the old URL structure.

**Comment by hharchani** [comment]
abey kyu? main/index.html will redirect to /index.html

**Pull request merged**